app/mailers/authentication_mailer.rb

```diff
@@ -11,6 +11,7 @@ def sign_up_email(candidate:, token:)
 
   def sign_in_email(candidate:, token:)
     @magic_link = candidate_interface_authenticate_url(magic_link_params(token))
+    @application_form = candidate.application_forms.order(:created_at).last
```

Comment:
i'm slightly uncomfortable with the fact that `Candidate#current_application` essentially does a `find_or_create`, but given that it exists, i think it makes sense to use it here. the worst possible outcome is that we create an `ApplicationForm` for a `Candidate`, which has to happen some time!

Reply:
ok great, I'll switch it to that.